apps/client/src/graphql/getChannelHeader.graphql

```diff
@@ -0,0 +1,13 @@
+query GetChannelHeader($channelId: Int!) {
```

Comment:
Can you update the name file with uppercase please, I have update the others files